### ISSUE TITLE
Curl 7.79.0 returns Unsupported response code in HTTP response

### ISSUE THREAD

**Issue by ocefpaf** [conda-forge/curl-feedstock#97]
Trying this with curl 7.79.0 fails:

```
> curl -k -s -v https://thredds.ucar.edu/thredds/dodsC/grib/NCEP/HRRR/CONUS_2p5km/Best                                                                                                                             ⏎

> User-Agent: curl/7.79.0
> Accept: */*
> 
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
* Unsupported response code in HTTP response
* Closing connection 0
* TLSv1.3 (OUT), TLS alert, close notify (256):
```

but passes with older versions of curl:

```
> User-Agent: curl/7.78.0
> Accept: */*
> 
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
* Mark bundle as not supporting multiuse
< HTTP/1.1 400 400
< Date: Wed, 22 Sep 2021 15:20:14 GMT
< Server: Apache
< X-Frame-Options: SAMEORIGIN
< X-Content-Type-Options: nosniff
< Strict-Transport-Security: max-age=63072000; includeSubdomains;
< Access-Control-Allow-Origin: *
< Content-Description: dods-error
< X-XSS-Protection: 1; mode=block
< Content-Security-Policy: form-action 'self'; frame-ancestors 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval'
< Connection: close
< Transfer-Encoding: chunked
< Content-Type: text/plain
< 
Error {
    code = 400;
    message = "Unrecognized request";
};
* Closing connection 0
* TLSv1.3 (IN), TLS alert, close notify (256):
* TLSv1.3 (OUT), TLS alert, close notify (256):
```

The 400 code is expected. Not sure if that is a problem that will be fixed by #96. Investigating...

**Comment by ocefpaf** [comment]
OK. It seems it is fixed in 7.79.1, let's remove the 7.79.0 to avoid headaches!

See https://github.com/curl/curl/issues/7738

**Comment by ocefpaf** [comment]
Fixed in https://github.com/conda-forge/admin-requests/pull/317
